src/common/constants/dev-config.json

```diff
@@ -44,10 +43,16 @@
   "dataBrowserYears": ["2023", "2022", "2021", "2020", "2019", "2018"],
   "dataBrowserMapsYears": ["2023", "2022", "2021", "2020", "2019", "2018"],
   "timedGuards": {
+    "2026": {
+      "Q1": "04/01/2026 - 05/30/2026 - 06/30/2026",
+      "Q2": "07/01/2026 - 08/29/2026 - 09/30/2026",
+      "Q3": "10/01/2026 - 11/30/2026 - 12/31/2026",
+      "annual": "01/01/2027 - 03/01/2027 - 12/31/2029"
+    },
     "2025": {
       "Q1": "04/01/2025 - 05/30/2025 - 06/30/2025",
       "Q2": "07/01/2025 - 08/29/2025 - 09/30/2025",
-      "Q3": "10/01/2025 - 11/29/2025 - 12/31/2025",
+      "Q3": "10/01/2025 - 12/01/2025 - 12/31/2025",
```

Comment:
If there is any questions reference ticket #2301 to see why the date is being changed.